packages/odyssey-react-mui/src/properties/translations/odyssey-react-mui_fr.properties

```diff
@@ -0,0 +1,7 @@
+field.error.screen.reader.text = Erreur
+field.label.optional.text = Optionnel
+severity.success = succès
+severity.warning = avertissement
+severity.info = info
+severity.error = erreur
+toast.close.text = fermé
```

Comment:
I will be deleting this file toward the end as that will be added by the translation partners